tooling/eslint/base.js

```diff
@@ -87,6 +87,7 @@ export default tseslint.config(
     files: ["**/*.spec.ts", "**/*.spec.tsx", "**/*.test.ts", "**/*.test.tsx"],
     rules: {
       "@typescript-eslint/no-unsafe-assignment": "off",
+      "@typescript-eslint/no-empty-function": "off",
```

Comment:
why do we allow empty functions? do the tests require this?

Reply:
some tests, especially skipped tests, will trigger it. No need for lint to be super strict on tests too